Classwork 3 19042024.R

```diff
@@ -1,19 +1,21 @@
+# To generate a vector of random numbers with a normal distribution:
 x <-rnorm(5)
 x
-
+# Another example on vector math
 N <-100000
 a <-rnorm(N)
 b <- rnorm(N)
 
 #Vectorized approach
 c <- a * b
 c
-
+# visualize the random numbers with normal distribution 
 X <- rnorm(5)
 X 
 plot(X)
 hist(X)
-

```

Comment:
# visualize the random numbers with normal distribution with n,mean,standard deviation
# multiple examples with the data